src/common/HeaderSearchButton.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { TouchableHighlight } from 'react-native';
+import { FontAwesome } from '@expo/vector-icons';
+import { useNavigation } from 'react-navigation-hooks';
+import styles from './styles';
+
+export default function SearchButton() {
+  const { push } = useNavigation();
+  return (
+    <TouchableHighlight onPress={() => push('Search')} style={styles.headerIcon}>
+      <FontAwesome
+        size={18}
+        name="search"
+        color="#999999"
```

Comment:
For colors in hex code, if we have repeated digits we may use a short hand.
For instance, `#114455` could be written as `#145`

```suggestion
        color="#999"
```